liota/lib/transports/web_socket.py

```diff
@@ -56,6 +56,7 @@ def connect_soc(self):
             self.WebSocketConnection(self.url, False)
         except Exception:
             log.exception("WebSocket exception, please check the WebSocket address and try again.")
+            raise Exception("WebSocket exception, please check the WebSocket address and try again.")
```

Comment:
Catching "Exception" is fine, but while raising an exception, we should raise a specific one like LiotaException or LiotaConnectionException. Anyway, for this particular case, we can just remove the try catch block completely, as we are just rethrowing the caught exception.

What happens when the exception is thrown? Will liota program stop running? For package, i guess it will not get loaded (not affecting liotad), correct?